src/components/Modal/src/Modal.vue

```diff
@@ -32,7 +32,7 @@
 
 	.Modal {
 		width: 600px;
-		max-height: calc(100vh - 48px);
+		max-height: calc(100vh - 64px);
```

Comment:
Add a comment here about the calc, I assume it's to make space for the actionbar? Or is this simply to provide vertical padding above / below modal on desktop?

Reply:
Nope, this has nothing to do for actionbar, this just so that the Modal has some 32px of margin above and below it if the modal content overflows the modal. I got these numbers from Shimaa.